xml/System.Collections.Generic/OrderedDictionary`2.xml

```diff
@@ -319,9 +319,9 @@
         <Parameter Name="value" Type="TValue" />
       </Parameters>
       <Docs>
-        <param name="key">To be added.</param>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="key">The object to use as the key of the element to add.</param>
```

Comment:
I can't comment on the exact line, but it looks like the summary and remarks for the type itself in lines 81-84 haven't been ported. Is this an issue with the porting tool potentially?

Reply:
Unsure. Are those properly commented in triple slash?

Reply:
It would seem so: https://github.com/dotnet/runtime/blob/9d7551ecd3e1b1b6f224eabecc1c7f91a41f259c/src/libraries/System.Collections/src/System/Collections/Generic/OrderedDictionary.cs#L18-L26

Reply:
That link (lines 18-26) point to the docs of "OrderedDictionary".

This xml file is for "OrderedDictionary<TKey, TValue>". 

It has the same problem anyway, it's documented in triple slash. https://github.com/dotnet/runtime/blob/9d7551ecd3e1b1b6f224eabecc1c7f91a41f259c/src/libraries/System.Collections/src/System/Collections/Generic/OrderedDictionary.cs#L62-L63

I need to figure out what's wrong. Might be the tool's problem.

Reply:
I wonder if this could be the root cause: https://github.com/dotnet/runtime/blob/9d7551ecd3e1b1b6f224eabecc1c7f91a41f259c/src/libraries/System.Collections/src/System/Collections/Generic/OrderedDictionary.cs#L29C1-L33C7

Do you know what determines if we make it `internal sealed`?

Reply:
The type is being multi-targeted to netstandard2.0 because it's a dependency in System.Text.Json

Reply:
Since this comment was unrelated to the PR, we can resolve it separately.